src/data/mainnet/tokens-info.json

```diff
@@ -416,7 +416,7 @@
     "symbol": "ETHIX"
   },
   {
-    "address": "0xf4361a8fbcf6b6a2402f1a1d6c4b945e59a29934",
+    "address": "0x27cd006548dF7C8c8e9fdc4A67fa05C2E3CA5CF9",
```

Comment:
The address needs to be lowercased for the test to pass.

Reply:
hi, we have done a migration, the old token is going to be deprecated and paused (noobdy will transact), and we have done already a balance migration to the new one

Reply:
Any issue updating the address to lowercase @garciapd? I can't merge the PR without it.

Reply:
done in a new commit :)